application/src/main/java/dev/ikm/komet/sampler/SamplerApp.java

```diff
@@ -0,0 +1,133 @@
+package dev.ikm.komet.sampler;
```

Comment:
Can we move the Sampler Application into the test area?
`application/src/test/dev/ikm/komet/sampler/`

Reply:
Done! I was having issues before on the "test" area but now have moved it to there and got it to work.